src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  &__star {
+    display: block;
+    width: 16px;
+    height: 16px;
```

Comment:
u can create a mixin for creating squares (f.e. for star and logo)

Reply:
good advice about mixin